tests/src/Functional/SettingsTest.php

```diff
@@ -78,6 +79,36 @@ public function testSettingsForm(): void {
     $url_field->setValue('www.drupal.org');
     $assert_session->buttonExists('Save configuration')->press();
     $assert_session->statusMessageContains('Manually entered paths should start with one of the following characters: / ? #', 'error');
+
+    // Test that the introduction text saved and preserved.
+    $this->drupalGet(Url::fromRoute('oe_subscriptions.settings'));
+    $introduction_text = $assert_session->fieldExists('Introduction text');
+    $introduction_text->setValue('Test text.');
+    $assert_session->buttonExists('Save configuration')->press();
+    $assert_session->statusMessageContains('The configuration options have been saved.', 'status');
+    $this->drupalGet(Url::fromRoute('oe_subscriptions.settings'));
+    $this->assertEquals($introduction_text->getValue(), 'Test text.');
```

Comment:
Always put first the expected value, and second the actual value. Please fix it in all this file.

Reply:
Done